src/Features/CSharp/Portable/InvokeDelegateWithConditionalAccess/InvokeDelegateWithConditionalAccessAnalyzer.cs

```diff
@@ -33,8 +33,16 @@ protected override void InitializeWorker(AnalysisContext context)
 
         private void SyntaxNodeAction(SyntaxNodeAnalysisContext syntaxContext)
         {
-            var options = syntaxContext.Options.GetOptionSet();
-            var styleOption = options.GetOption(CSharpCodeStyleOptions.PreferConditionalDelegateCall);
+            var options = syntaxContext.Options;
```

Comment:
This would be addressed in PR [https://github.com/dotnet/roslyn/pull/15029](https://github.com/dotnet/roslyn/pull/15029) but moving the options down a layer would require significant changes to how the analyzer driver works.

---
In reply to: [87468049](https://github.com/dotnet/roslyn/pull/15065#discussion_r87468049) [](ancestors = 87468049)